src/WebJobs.Extensions.DurableTask/Correlation/TraceContextBaseExtensions.cs

```diff
@@ -24,7 +24,14 @@ public static RequestTelemetry CreateRequestTelemetry(this TraceContextBase cont
             telemetry.Timestamp = context.StartTime;
             telemetry.Id = context.TelemetryId;
             telemetry.Context.Operation.Id = context.TelemetryContextOperationId;
+
+            // Operation Name is set as "DtActivity FunctionName" and we only want the "FunctionName" as the operationName
```

Comment:
Are there any other formats of OperationName? I know that we don't have full parity yet with all our APIs, but do we have any operations for things like Timers or External Events?

Reply:
No, I don't see any other formats for operation name. The operation name for "DtActivity FunctionName" is set [here](https://github.com/Azure/durabletask/blob/bd6186992fd0d55372d0f0989f1632efd2ff72d1/src/DurableTask.AzureStorage/AzureStorageOrchestrationService.cs#L1443).

I don't see timers or external events getting tracked. So far, distributed tracing tracks the events listed [here](https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/TraceConstants.cs).

Reply:
I think Timers/External events are still missing scenarios that we may want to capture for GA. I think this is sufficient for now.